### DISCUSSION TITLE
Content Template - page.url contains

### DISCUSSION THREAD

**Discussion by LuchianC** [coddingtonbear/obsidian-web#161] [Q&A] [answered]
There is any solution to use contains helper? Because i want to configure Content Template somthing like that:

{{#contains youtube 'page.url' }}
#youtube
{{/contains}}

Thanks,

**Comment by LuchianC** [comment, marked as answer]
It's ok, i made a new template for youtube 😄

**Comment by TheUdK** [comment]
@LuchianC , would you care to share?   👀

**Reply by LuchianC** [reply]
Sure, it's something like this:
```
---
page-title: {{json page.title}}
url: {{page.url}}
date: "{{date}}"
duration: 
tags: youtube noTag
---
#VideoToWatch [[VideoToWatch]]
`#VideoWatched [[VideoWatched]]

[link]({{page.url}})
```
